props/saber_sabersense_buttons.h

```diff
@@ -0,0 +1,1235 @@
+/*
+============================================================
+================== SABERSENSE™ PROP FILE ===================
+============================================================
+
+Built on SA22C programming by Matthew McGeary.
+Modified by Chris Carter with substantial support and 
+contributions from Fredrik Hubinette and Brian Conner.
+
+This prop file references certain custom sound files 
+to aid in saber function navigation. These sound files 
+are optional and are available as a free download from:
+https://sabersense.square.site/downloads
+
+
+============================================================
+========= BUTTON SYSTEM PRINCIPLES AND LOGIC NOTES ========= 
+
+The Sabersense™ button control system has been engineered 
```

Comment:
C++ files are generally supposed to be plain ASCII.
To avoid problems either remove the TM character, or replace it with "(TM)"